src/dialect/mysql/mysql-query-compiler.ts

```diff
@@ -72,4 +73,13 @@ export class MysqlQueryCompiler extends DefaultQueryCompiler {
       this.visitNode(node.where)
     }
   }
+
+  protected override visitOrConflict(node: OrConflictNode): void {
+    if (node.resolution === 'Ignore') {
+      this.append(' ')
```

Comment:
Implemented the override for ignore per the linked issue, but I left the other calls alone. Should I also override them for consistency?

Reply:
See other comments.